rvsol/src/RISCV.sol

```diff
@@ -888,7 +888,7 @@ contract RISCV is IBigStepper {
 
                     let errCode := 0
                     // ensure MAP_ANONYMOUS is set and fd == -1
-                    switch or(iszero(and(flags, 0x20)), not(eq(fd, u64Mask())))
+                    switch or(iszero(and(flags, 0x20)), eq(eq(fd, u64Mask()), 0))
```

Comment:
Can we use `iszero` for consistency?

```suggestion
                    switch or(iszero(and(flags, 0x20)), iszero(eq(fd, u64Mask())))
```

Reply:
Updated here! https://github.com/ethereum-optimism/asterisc/pull/113/commits/e5a584226fab1b0c6c2aae4ffebf2a5e7cd8c60b Thanks :)